docs/quickstart.rst

```diff
@@ -1,6 +1,68 @@
 Quickstart
 ==========
 
+Use *septentrion* as your migration tool in 2 easy steps:
+  1. Write your migrations.
+  2. Run your migrations.
+
+Write migrations
+----------------
+Your migration folder should be organised as follow:
+ - each version has its own folder;
+ - in a version folder, migrations will be executed in alphabetical order;
+ - `schemas` folder contains state of the schema at a given version.
+
+# I'm not sure if the explanation around the naming convention makes sense in a quick start, probably not?
```

Comment:
Not sure either, especially given we're not using the convention so far. When Septentrion starts doing something with the names, it might be interesting to introduce the convention?

Let's keep it simple :)